src/image.py

```diff
@@ -143,12 +145,9 @@ def build(self):
         self.update_pre_build_configuration()
 
         # Start building the image
-        if self.context:
-            with open(self.context.context_path, "rb") as context_file:
-                self.docker_build(fileobj=context_file, custom_context=True)
-                self.context.remove()
-        else:
-            self.docker_build()
+        with open(self.context.context_path, "rb") as context_file:
+            self.docker_build(fileobj=context_file, custom_context=True)
+            self.context.remove()
```

Comment:
Please undo this change, because we might have images being built in the future that do not have any files to be included through build context, even if we do not right now.